packages/runtime/test-runtime-utils/api-report/test-runtime-utils.api.md

```diff
@@ -460,7 +461,7 @@ export class MockFluidDataStoreRuntime extends EventEmitter implements IFluidDat
     get local(): boolean;
     set local(local: boolean);
     // (undocumented)
```

Comment:
We should add docs to this. Is there a good `{@inheritDoc}` target?

Reply:
The corresponding member(s) in IFluidDataStoreRuntime also lack docs 😅 . If it was an obvious one I'd add them but I recall some complexity around "this is a logger that you as consumer of this interface can use" vs "this is a logger that implementations of the interface should all have for internal purposes" so I'll leave this be for now.